sky-uk/ruby-appdynamics-buildpack
# Stop evaluating in EOF and sed log4j

**To Test**

There's a review app of HelpSite that uses this branch: https://dashboard.heroku.com/apps/test-help-site-pr-1014/

Tailing the logs of this app should show you the increase / decrease in log levels

- With `AD_LOG_LEVEL` set to `info` you should get the same amount of logging as production help-site
- With `AD_LOG_LEVEL` set to `warn` you'll see a reduced amount of logging per request as compared to production help-site
- With `AD_LOG_LEVEL` blank it should default to `warn` and you'll see a reduced amount of logging per request as compared to production help-site
